Projects/Features/Scene/GroupDetailScene/MemberList/MemberListView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  MemberListView.swift
+//  GroupDetail
+//
+//  Created by 최혜린 on 6/22/24.
+//  Copyright © 2024 com.mashup.gabbangzip. All rights reserved.
+//
+
+import ComposableArchitecture
+import SwiftUI
+
+import DesignSystem
+import Models
+
+public struct MemberListView: View {
+  let store: StoreOf<MemberListCore>
+
+  public init(store: StoreOf<MemberListCore>) {
+    self.store = store
+  }
+
+  public var body: some View {
+    VStack(spacing: 0) {
+      NavigationBar(
+        type: .titleWithBackButton("그룹원"),
+        backButtonAction: {
+          store.send(.backButtonDidTap)
```

Comment:
```suggestion
          store.send(.backButtonTapped)
```
이건 컨벤션 문제인데 버튼 액션에서 Tapped 혹은 DidTap을 자주 사용하는데 여기서 의견을 받아 통일시키면 어떨까요
저는 Tapped가 더 익숙합니다 @GREENOVER  @serena0720

Reply:
저도 tapped 스타일로 쓰긴 합니다~
여러분 편하신대로!

Reply:
이전 PR에서 ~~ButtonTapped로 구현해놔서 이쪽으로 통일하는 거에 동의합니다!

Reply:
넵 그럼 Tapped로 통일하도록 하겠습니다~!